tests/specs/spa/interaction-tracking.e2e.js

```diff
@@ -1,8 +1,8 @@
 import { notIE } from '../../../tools/browser-matcher/common-matchers.mjs'
 
-describe('interaction tracking works inside', () => {
-  it.withBrowsersMatching(notIE)('single fetch with formData', async () => {
-    const url = await browser.testHandle.assetURL('spa/fetch-formdata-onclick.html')
+describe('interaction tracking', () => {
+  it.withBrowsersMatching(notIE)('works with Request formData', async () => {
```

Comment:
can we keep a note somewhere as to why IE is excluded? I am assuming its due to fetch being invoked by formData() after looking through it and by what the test was named originally

Reply:
IE doesn't support fetch